src/main/java/com/avail/utility/Strings.kt

```diff
@@ -167,6 +167,21 @@ object Strings {
 		}
 	}
 
+	/**
+	 * Answer a [String] consisting of [count] repetitions of [string],
+	 * concatenated.
+	 */
+	@JvmStatic
+	fun repeated (string: String, count: Int): String
+	{
+		assert(count >= 0)
+		return buildString(string.length * count) {
+			repeat(count) {
+				append(string)
+			}
+		}
```

Comment:
Wow, sweet implementation. I didn't know about `buildString`. Is that Kotlin, or is it us?

Reply:
Technically... both.  I had the idea a couple of months ago, used it for a while, then discovered the Kotlin library had a method with the same name and signature!